src/zui/components/ZUIAvatarGroup/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { Box, Typography, useTheme } from '@mui/material';
+import { FC } from 'react';
+
+import ZUIAvatar from '../ZUIAvatar';
+import { ZUISize } from '../types';
+
+type ZUIAvatarGroupProps = {
+  /**
+   * List of the people you want to display as avatars.
+   */
+  avatars: {
+    firstName: string;
+    id: number;
+    lastName: string;
+  }[];
+
+  /**
+   * Maximum number of avatars shown.
+   */
+  max?: number;
+
+  /**
+   * The size of the avatars. Defaults to 'medium'.
+   */
+  size?: ZUISize;
+
+  /**
+   * The shape of the avatars. Defaults to 'circular.
+   */
+  variant?: 'circular' | 'square';
+};
+
+const ZUIAvatarGroup: FC<ZUIAvatarGroupProps> = ({
+  avatars,
+  max,
+  size = 'medium',
+  variant = 'circular',
+}) => {
+  const theme = useTheme();
+
+  let avatarSize = '2rem';
+  let fontSize = '0.875rem';
+  if (size == 'small') {
+    avatarSize = '1.5rem';
+    fontSize = '0.625rem';
+  } else if (size == 'large') {
+    avatarSize = '3rem';
+    fontSize = '1rem';
+  }
+
+  const showOverflowNumber = max && max < avatars.length;
+  return (
+    <Box display="flex" gap="0.25rem">
+      {avatars.map((avatar, index) => {
+        if (showOverflowNumber && index > max - 2) {
+          return;
+        }
+        return (
+          <ZUIAvatar
+            key={avatar.id}
+            firstName={avatar.firstName}
+            id={avatar.id}
+            lastName={avatar.lastName}
+            size={size}
+            variant={variant}
+          />
+        );
+      })}
+      {showOverflowNumber && (
```

Comment:
The same thing happens here. When `showOverflowNumber == 0`, this expression evaluates to "0". React is programmed to ignore `undefined`, `null` and `false` nodes, but it will not ignore a `0` node (because it could be intentional to render a zero). So this expression renders a zero:

![image](https://github.com/user-attachments/assets/45601e62-7a6a-4333-808e-8f337411b2c4)

Reply:
Thank you for describing this! One of those things that always slip my mind! will add `!!` to ensure booleanity :)

Reply:
Fixed in 7fc52464ca1c9a89e07632ac919d95378bd21411.